src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -274,9 +279,10 @@ public static void makeGenotypeCall(final int ploidy,
                 gb.alleles(noCallAlleles(ploidy)).noGQ();
             } else {
                 final int maxLikelihoodIndex = MathUtils.maxElementIndex(genotypeLikelihoods);
-                final List<Allele> maxLikelihoodAlleles = GenotypeLikelihoods.getAlleles(maxLikelihoodIndex, ploidy).stream()
-                        .map(allelesToUse::get).collect(Collectors.toList());
-                gb.alleles(maxLikelihoodAlleles);
+                final GenotypeLikelihoodCalculator glCalc = GL_CALCS.getInstance(ploidy, allelesToUse.size());
+                final GenotypeAlleleCounts alleleCounts = glCalc.genotypeAlleleCountsAt(maxLikelihoodIndex);
```

Comment:
Yes, confirmed.  Also, it appears to fix a bug because the htsjdk code had a hard-wired assumption that diploid == biallelic.